packages/next-tinacms-github/src/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const ACCESS_TOKEN_KEY = 'github_access_token'
+export const FORK_KEY = 'fork_full_name'
+export const HEAD_BRANCH_KEY = 'head_branch'
```

Comment:
What if these were defined in `react-tinacms-github` and exported from there?

Reply:
makes sense to me. it would mean introducing a hard dependency to `react-tinacms-github` but at this point I'm not sure that's really a concern

Reply:
True. I'm good with magic variables temporarily being the same by "coincidence". 

That sounds much worse after I write it but I stand by that statement